process_request/helpers.py

```diff
@@ -83,26 +83,30 @@ def get_instance_data(instance_list):
             resolved top containers and digital objects.
 
     Returns:
-        tuple: a tuple containing instance type, indicator, location, and
-            barcode for the instance.
+        tuple: a tuple containing instance type, indicator, location,
+            barcode, and container ref for the instance.
```

Comment:
This is _very_ minor, but the "barcode" field is either the top container barcode or the Digital Object ID; does it make sense to clarify in the docstring?